src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -82,6 +82,33 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
+{{ if OryxAppInsightInjectCommand | IsNotBlank }}
+echo
+echo "Preparing Envrionment to set up application insight..."
+echo "Running '{{ OryxAppInsightInjectCommand }}'..."
+echo
+{{ OryxAppInsightInjectCommand }}
+echo
+echo "Creating application insight loader file to '$DESTINATION_DIR'"
+echo "// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+var appInsights = require('applicationinsights');
+
+if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+    try {
+        appInsights
+            .setup()
+            .start();
+    } catch (e) {
+        console.error(e);
+    }
+}">oryxappinsightloader.js
```

Comment:
this constant name is used in both C# and Go, so it would be a good fit for `build-constants.yaml`

Reply:
I will create a seperate PR for this, I don't want to do this in this PR